generators/vsix-vs-win/Dialogs Template/DialogsTemplate/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("DialogsTemplate")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Cecilia Avila")]
```

Comment:
Remember to update the `Company Name` field before promoting into Microsoft's repo. This can be changed in the `vsix.manifest` file.